crates/revmc-worker/src/external.rs

```diff
@@ -1,51 +1,70 @@
-use std::sync::{Arc, RwLock};
+use std::{
+    fmt::{Debug, Display},
+    num::NonZeroUsize,
+    sync::{Arc, RwLock},
+};
 
 use crate::{
     error::ExtError,
     worker::{aot_store_path, CompileWorker, SledDB},
 };
 use alloy_primitives::B256;
+use lru::LruCache;
 use once_cell::sync::OnceCell;
-use revm_primitives::SpecId;
+use revm::Database;
+use revm_primitives::{AccessListItem, SpecId};
 use revmc::EvmCompilerFn;
 
 pub(crate) static SLED_DB: OnceCell<Arc<RwLock<SledDB<B256>>>> = OnceCell::new();
 
 #[derive(Debug)]
-pub struct EXTCompileWorker {
+pub struct EXTCompileWorker<DB> {
     compile_worker: Box<CompileWorker>,
+    pub cache: LruCache<String, (EvmCompilerFn, libloading::Library)>,
```

Comment:
can we use B256(Fixed Bytes) keys instead of String?

Reply:
and Library value seems be not used. so can we only use value of `EvmCompilerFn`

```
    pub cache: LruCache<String, EvmCompilerFn>,
```

Reply:
EvmCompilerFn will result to segfauilt, if we dont hold library

Reply:
> can we use B256(Fixed Bytes) keys instead of String? if possible, we don't need cast to String type

done